README.md

```diff
@@ -6,6 +6,20 @@
 # platform-services-registry
 Platform services OCP project registry
 
+## Callbacks
+export SSO_TOKEN=YOUR_TOKEN_HERE
+export SSO_CLIENT_ID=YOUR_CLIENT_ID_HERE
+export SSO_TOKEN_URL=YOUR_TOKEN_URL_HERE
+
+curl -sX POST -H "Content-Type: application/x-www-form-urlencoded" \
+    --data "grant_type=client_credentials&client_id=${SSO_CLIENT_ID}&client_secret=${SSO_TOKEN}" \
+    ${SSO_TOKEN_URL} | \
+    jq '.access_token' |
+    cut -d "\"" -f 2
+
+Then use a callback:
+curl -vX PUT -H "Content-Type: application/json" -H "Authorization: Bearer ${MYT}" http://localhost:8100/api/v1/provision/1/namespace -d @data.json
```

Comment:
instruction missing a step to set token value to var MYT

Reply:
Those are just notes in the README. I'm currently working through better documentation.